src/app/pages/apps/components/installed-apps/app-info-card/app-info-card.component.html

```diff
@@ -103,16 +103,17 @@ <h3 mat-card-title>
             {{ app().metadata.train | orNotAvailable }}
           </div>
         </div>
-        @if ((app().portals | keyvalue).length > 0 ) {
-          <div class="portals">
-            @for (portal of app().portals | keyvalue; track portal.key) {
+        @if (sortedPortals().length > 0) {
+          <div class="portals" [class.has-many]="sortedPortals().length > 1">
+            @for (portal of sortedPortals(); track portal[0]) {
```

Comment:
I think dealing with indices instead of named attributes makes the code less readable